pkg/licensing/classifier.go

```diff
@@ -96,13 +97,26 @@ func googleClassifierLicense(filePath string, contents []byte) types.LicenseFile
 		}
 		licenseLink := fmt.Sprintf("https://spdx.org/licenses/%s.html", m.Name)
 
-		findings = append(findings, types.LicenseFinding{
+		finding := types.LicenseFinding{
 			Name:       m.Name,
 			Confidence: m.Confidence,
 			Link:       licenseLink,
-		})
+		}
+
+		if _, ok := findingsMap[finding]; !ok {
+			findingsMap[finding] = true
+		}
```

Comment:
I'd like to know the reason we should check the key existence. Doesn't the following work?

```suggestion
		findingsMap[finding] = true
```

Reply:
no reason, it works fine. changed